Src/AutoFixture/DataAnnotations/TimeSpanRangedRequestRelay.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Globalization;
+using AutoFixture.Kernel;
+
+namespace AutoFixture.DataAnnotations
+{
+    /// <summary>
+    /// Handles <see cref="RangedRequest"/> of TimeSpan type by forwarding requests
+    /// to the <see cref="RangedNumberRequest"/> with min and max TimeSpan as milliseconds values.
+    /// </summary>
+    public class TimeSpanRangedRequestRelay : ISpecimenBuilder
+    {
+        /// <inheritdoc />   
+        public object Create(object request, ISpecimenContext context)
+        {
+            if (context == null) throw new ArgumentNullException(nameof(context));
+
+            var rangedRequest = request as RangedRequest;
+
+            if (rangedRequest == null)
+                return new NoSpecimen();
+
+            if (rangedRequest.MemberType != typeof(TimeSpan))
+                return new NoSpecimen();
+
+            return CreateRangedTimeSpanSpecimen(rangedRequest, context);
+        }
+
+        private static object CreateRangedTimeSpanSpecimen(RangedRequest rangedRequest, ISpecimenContext context)
+        {
+            if (!(rangedRequest.Minimum is string) || !(rangedRequest.Maximum is string)) return new NoSpecimen();
```

Comment:
To follow the same styling please add line break (use one line for guard clauses only).